services/apps/alcs/src/alcs/notice-of-intent/notice-of-intent.entity.ts

```diff
@@ -0,0 +1,44 @@
+import { Type } from 'class-transformer';
+import { Column, Entity, JoinColumn, ManyToOne, OneToOne } from 'typeorm';
+import { ApplicationLocalGovernment } from '../application/application-code/application-local-government/application-local-government.entity';
+import { Card } from '../card/card.entity';
+import { ApplicationRegion } from '../code/application-code/application-region/application-region.entity';
+import { Base } from '../../common/entities/base.entity';
+
+@Entity()
+export class NoticeOfIntent extends Base {
+  constructor(data?: Partial<NoticeOfIntent>) {
+    super();
+    if (data) {
+      Object.assign(this, data);
+    }
+  }
+
+  @Column({ unique: true })
+  fileNumber: string;
+
+  @Column()
+  applicant: string;
+
+  @Column({ type: 'uuid' })
+  cardUuid: string;
```

Comment:
I wonder if card needs to be nullable the same way as in application. So when we move things from OATS we will not be creating cards for all the data?